src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -48,6 +48,23 @@ protected function tearDown()
         }
     }
 
+    /**
+     * @group legacy
+     * @expectedDeprecation The provided cwd does not exist. Command is currently ran against getcwd(). This behaviour is deprecated since version 3.4 and will be removed in 4.0.
+     */
+    public function testInvalidCwd()
+    {
+        // Check that it works fine if the CWD exists
+        $cmd = new Process('echo test > testing1.txt', __DIR__);
+        $this->assertEquals(0, $cmd->run());
```

Comment:
should be `assertSame` - but to me this assertion is not needed: the fact that a process runs correctly when all is fine is already tested in other cases

Reply:
Done